pom.xml

```diff
@@ -130,8 +130,21 @@
         <surefire.jvm.args.java-version></surefire.jvm.args.java-version>
         <surefire.jvm.args.jacoco></surefire.jvm.args.jacoco>
         <failsafe.jvm.args.jacoco></failsafe.jvm.args.jacoco>
+        <!--
+             Pass these properties as system properties to be able to switch to
+             a different database without re-compiling the hibernate.properties file
+             (which is located in a different file)
+         -->
+        <failsafe.jvm.args.hibernate-orm>
+            -Dhibernate.dialect=${db.dialect}
+            -Dhibernate.connection.driver_class=${jdbc.driver}
+            -Dhibernate.connection.url=${jdbc.url}
+            -Dhibernate.connection.username=${jdbc.user}
+            -Dhibernate.connection.password=${jdbc.pass}
+            -Dhibernate.connection.isolation=${jdbc.isolation}
+        </failsafe.jvm.args.hibernate-orm>
         <surefire.jvm.args>${surefire.jvm.args.memory} ${surefire.jvm.args.java-version} ${surefire.jvm.args.jacoco}</surefire.jvm.args>
```

Comment:
You don't expect to need these properties for surefire?

Reply:
As discussed, no, I wouldn't expect to need them in surefire tests, since surefire tests are unit tests, and a test with Hibernate ORM is (by definition) an integration test.